code/modules/jobs/job_types/clown.dm

```diff
@@ -22,6 +22,13 @@
 /datum/job/clown/after_spawn(mob/living/carbon/human/H, mob/M)
 	. = ..()
 	H.apply_pref_name("clown", M.client)
+	if(HAS_TRAIT(SSstation, STATION_TRAIT_BANANIUM_SHIPMENTS))
+		var/obj/structure/closet/supplypod/centcompod/toLaunch = new()
```

Comment:
Isn't there a clown supplypod?

Reply:
i dont know i saw a style that wasnt functionally in idk if i missed something

Reply:
???
toLaunch.setStyle(STYLE_HONK)
sir....

Reply:
![image](https://user-images.githubusercontent.com/29008542/105443849-b9344080-5c3a-11eb-90de-fd3558748933.png)

Reply:
![image](https://user-images.githubusercontent.com/20987591/105530428-10dcb580-5ce8-11eb-9f6e-c705fd1ebb6d.png)

Reply:
yes and
trust the stylesystem sir